src/ethernet/ip_device.cpp

```diff
@@ -361,11 +361,6 @@ void ip_device::inject_frames_loop(std::shared_ptr<rs_rtp_stream> rtp_stream)
         rtp_stream.get()->is_enabled = true;
 
         rtp_stream.get()->frame_data_buff.frame_number = 0;
-
-        /// FIXME: bbp and stride should be filled by m_rs_stream. but There are something wrong in m_rs_stream.
```

Comment:
Please keep this workaround until fix get_stride_in_bytes and get_bytes_per_pixel for Y8 image.

Reply:
I totally agree with you, but there is one issue. We are trying to keep the network extensions library "external" to the main librealsense code. It means we cannot use "librealsense" namespace and we do use include files from the "src" directory. I will think about the solution for this issue. Any suggestions are welcome.

Reply:
Got it. Thank you.

Reply:
I think, these code should be moved to rs_rtp_stream::rs_rtp_stream() method. <librealsense>/src/ethernet/rs_rtp_stream.hh